app/navigation/Navigator.scala

```diff
@@ -126,6 +126,10 @@ class Navigator @Inject()() extends BaseNavigator {
       (userAnswers: UserAnswers) => routes.ConfirmationController.onPageLoad(userAnswers.ern, userAnswers.arc)
     case CheckAnswersItemPage(_) =>
       (userAnswers: UserAnswers) => routes.AddedItemsController.onPageLoad(userAnswers.ern, userAnswers.arc)
+    case AddedItemsPage =>
+      (userAnswers: UserAnswers) => routes.AddMoreInformationController.loadMoreInformation(userAnswers.ern, userAnswers.arc, NormalMode)
+    case RemoveItemPage(_) =>
+      (userAnswers: UserAnswers) => routes.AddedItemsController.onPageLoad(userAnswers.ern, userAnswers.arc)
```

Comment:
Does this need to check how many items are left in the list? (in the scenario where they've removed the last item). As, if it's empty, should it redirect to the SelectItemsView?